api/status/codes.go

```diff
@@ -0,0 +1,26 @@
+package status
+
+// All supported status codes.
+const (
+	OK                        = 0
```

Comment:
have you thought about some helper bit operations here? just to describe why it is 1024, 1025, 1026, 1027 and then 2048

Reply:
this is not needed, code is just a code, nobody cares what number is it